lib/core/application/event/create_event_bloc/create_event_bloc.dart

```diff
@@ -151,16 +124,16 @@ class CreateEventBloc extends Bloc<CreateEventEvent, CreateEventState> {
     if (state.isValid) {
       emit(state.copyWith(status: FormzSubmissionStatus.inProgress));
       final result = await _eventRepository.createEvent(
-        input: CreateEventInput(
+        input: Input$EventInput(
           title: title.value,
           description: title.value,
           private: false,
           verify: false,
-          start: DateTime.parse(state.start.value!.toUtc().toIso8601String()),
-          end: DateTime.parse(state.end.value!.toUtc().toIso8601String()),
+          start: DateTime.parse(event.start.toUtc().toIso8601String()),
+          end: DateTime.parse(event.end.toUtc().toIso8601String()),
           timezone: "Asia/Bangkok",
-          guestLimit: 100,
-          guestLimitPer: 2,
+          guest_limit: 100,
```

Comment:
Can we use camelCase ?

Reply:
meaning that we cannot convert this to camelCase ? so in the future, we have to use snake_case for all properties ?